Formula/b/bash.rb

```diff
@@ -112,10 +112,59 @@ def install
     # Homebrew's bash instead of /bin/bash.
     ENV.append_to_cflags "-DSSH_SOURCE_BASHRC"
 
+    # Amend the default value of `BASH_LOADABLES_PATH` to include Homebrew's
+    # Bash installation path.
+    #
+    # See: https://www.gnu.org/software/bash/manual/html_node/Bash-Builtins.html#index-enable
+    # See: http://git.savannah.gnu.org/cgit/bash.git/tree/config-top.h#n80
+    #
+    # With this enabled one can now do this:
+    #
+    # ```bash
+    # $ type csv
+    # -bash: type: csv: not found
+    # $ enable csv
+    # $ type csv
+    # csv is a shell builtin
+    # $ help csv
+    # csv: csv [-a ARRAY] string
+    #     Read comma-separated fields from a string.
+    #
+    #     Parse STRING, a line of comma-separated values, into individual fields,
+    #     and store them into the indexed array ARRAYNAME starting at index 0.
+    #     If ARRAYNAME is not supplied, "CSV" is the default array name.
+    # $ csv 'a,"b",c'
+    # $ declare -p CSV
+    # declare -a CSV=([0]="a" [1]="b" [2]="c")
+    # $ enable -n csv
+    # $ type csv
+    # -bash: type: csv: not found
+    # ```
+    #
+    bash_loadables_path=[
+      "#{lib}/bash",
+      # Stock Bash paths; keep them for backwards compatibility.
+      "/usr/local/lib/bash",
+      "/usr/lib/bash",
+      "/opt/local/lib/bash",
+      "/usr/pkg/lib/bash",
+      "/opt/pkg/lib/bash",
+      ".",
+    ].join(":")
+    ENV.append_to_cflags "-DDEFAULT_LOADABLE_BUILTINS_PATH='\"#{bash_loadables_path}\"'"
+
     system "./configure", "--prefix=#{prefix}"
     system "make", "install"
   end
 
+  def caveats
+    <<~EOS
+      #{lib}/bash has been added to $BASH_LOADABLES_PATH.
+      For more info, see:
+        https://www.gnu.org/software/bash/manual/html_node/Bash-Builtins.html#index-enable
+    EOS
+  end
+
```

Comment:
```suggestion

```
I don't think we need this.

Reply:
The reason I added it was for discoverability of the feature. It's intended for people like my past self that would have been interested in this information. And it's not a noisy addition either — the user only sees it once when they install Bash or if they're specifically interested in information about the formula (via `brew info`). But it's also a customization specific to Homebrew Bash, so maybe it should be surfaced/presented somewhere.

Having said that, I don't mind removing it. Let me know what you think.

Reply:
It's a bit of a catch-22 in my opinion, but I'll remove it.